README.md

```diff
@@ -262,7 +264,7 @@ Joins the two parts using exactly one separator. If a separator is present at th
 
 Thank you for using *urlcat*!
 
-If you need any help using this library, feel free to [create a GitHub issue](https://github.com/balazsbotond/urlcat/issues/new/choose) and ask your questions. I'll try to answer as quickly as possible.
+For any kind of help needed while using this library, feel free to [create a GitHub issue](https://github.com/balazsbotond/urlcat/issues/new/choose), and ask your questions. I'll try to answer as quickly as possible.
```

Comment:
I think we should keep the second person voice here. It's more personal and directly addresses the reader.